shared/constants/types/chat2/index.tsx

```diff
@@ -115,6 +115,12 @@ export type UserReacjis = {
   skinTone: number
 }
 
+export type Coordinate = {
+  accuracy: number
+  lat: number
+  lon: number
+}
```

Comment:
Make it into a Record? This hasn't been converted to immer yet.

Reply:
What do I get from doing that? I'm just feeding new versions of these into the store on every update.

Reply:
We've had bugs before when using a mix of immutable types and regular JS objects, so just wanted to keep it consistent. Also with immutable you can actually check equality in the reducer to use the same object reference when the new location is the same. Probably won't make much difference though.